selenium.js

```diff
@@ -12,26 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const {
+import {
   Browser,
   Builder,
   By,
   Capabilities,
   Capability,
   logging,
   until
-} = require('selenium-webdriver');
-const bcdBrowsers = require('@mdn/browser-compat-data').browsers;
-const compareVersions = require('compare-versions');
-const fetch = require('node-fetch');
-const fs = require('fs-extra');
-const path = require('path');
-const chalk = require('chalk');
-const {Listr} = require('listr2');
-
-const secrets = require('./secrets.json');
-
-const resultsDir = path.join(__dirname, '..', 'mdn-bcd-results');
+} from 'selenium-webdriver';
+import bcd from '@mdn/browser-compat-data';
+const bcdBrowsers = bcd.browsers;
+import compareVersions from 'compare-versions';
+import fetch from 'node-fetch';
+import esMain from 'es-main';
+import fs from 'fs-extra';
+import path from 'path';
+import chalk from 'chalk';
+import listr2 from 'listr2';
+const {Listr} = listr2;
```

Comment:
Can't this be simplified into one line somehow? Has listr2 done something unusual with their exports?